src/ddmd/traits.d

```diff
@@ -370,6 +373,73 @@ extern (C++) Expression pointerBitmap(TraitsExp e)
     return ale;
 }
 
+extern (C++) Expression externDmangle(TraitsExp e, Scope* sc)
```

Comment:
> What does this function do? Why is it never executed?

It is used in core.internal.traits.externDFunc (https://github.com/dlang/druntime/pull/1830/files#diff-f1322962163674c1ee8c7a018193ab45R94), but that's actually never evaluated due to a typo. It used to fail without externDmangle, but I guess usage of externDFunc has changed to symbols that don't benefit from compression.

I'll add a comment and a test to the dmd test suite.